code/script.js

```diff
@@ -1,25 +1,70 @@
 // All the DOM selectors stored as short variables
+
 const chat = document.getElementById('chat')
+const nameInput = document.getElementById('name-input')
+const form = document.getElementById('name-form')
+const main = document.getElementById('main')
+const title = document.getElementById('title')
+const title2 = document.getElementById('title2')
+
+// The below const are containing wrapped HTML buttons and select tags. 
+
+const btnSelectMain = `
+  <button id='sushi' type='submit'>Sushi 🍣</button>
+  <button id='pokebowl' type='submit'>Pokebowl 🥗</button>
```

Comment:
On jessica pc the icon for the pokebowl is showing but on Marias mac it dosent (its only showing ?? there the icone would ne)